assets/css/_vehicle_properties_panel.scss

```diff
@@ -37,6 +37,34 @@
   @include font-header;
 }
 
+.m-vehicle-properties-panel__schedule-adherence {
+  @include font-small;
+  color: $color-font-dark;
+  display: flex;
+  margin-top: 0.7rem;
+}
+
+.m-vehicle-properties-panel__schedule-adherence-status-icon {
+  margin-right: 0.4rem;
+  stroke: none;
```

Comment:
`stroke: none` isn't needed since it's the default.
Above, `color: $color-font-dark` isn't needed since it's inherited.

Reply:
Removed the stroke. The color is needed, it falls back to black without it.

Reply:
Oh, oops, the color should be specified on `body` in `app.scss`, and then it won't be needed here. All the text is black right now.

Reply:
Ah yeah, that's better—moved.